── 1. Failure: Failure:1 (@tests.R#12)  ────────────────────────────────────────
Failure has been forced

── 2. Failure: Failure:2a (@tests.R#16)  ───────────────────────────────────────
Failure has been forced

── 3. Failure: Failure:2b (@tests.R#19)  ───────────────────────────────────────
FALSE isn't true.

── 4. Failure: Failure:loop (@tests.R#24)  ─────────────────────────────────────
`i` not equal to 2.
1/1 mismatches
[1] 1 - 2 == -1

── 5. Error: Error:1 (@tests.R#33)  ────────────────────────────────────────────
Message: stop
Backtrace:
 1. NULL
 2. base::eval(code, test_env)

── 6. Error: Error:3 (@tests.R#47)  ────────────────────────────────────────────
Message: !
Backtrace:
 1. f()
 2. g()
 3. h()

── 7. Error: Recursion:1 (@tests.R#56)  ────────────────────────────────────────
Message: This is deep
Backtrace:
  1. f(25)
  2. f(x - 1)
  3. f(x - 1)
  4. f(x - 1)
  5. f(x - 1)
  6. f(x - 1)
  7. f(x - 1)
  8. f(x - 1)
  9. f(x - 1)
 10. f(x - 1)
 11. f(x - 1)
 12. f(x - 1)
 13. f(x - 1)
 14. f(x - 1)
 15. f(x - 1)
 16. f(x - 1)
 17. f(x - 1)
 18. f(x - 1)
 19. f(x - 1)
 20. f(x - 1)
 21. f(x - 1)
 22. f(x - 1)
 23. f(x - 1)
 24. f(x - 1)
 25. f(x - 1)
 26. f(x - 1)

── 8. Error: errors thrown at block level are entraced (@tests.R#99)  ──────────
Message: foo
Backtrace:
 1. f()
 2. g()

── 9. Error: errors thrown from a quasi-labelled argument are entraced (@tests.R
Message: foo
Backtrace:
 1. testthat::expect_is(foo(), "foo")
 4. foo()

── 10. Error: errors thrown from a quasi-labelled argument are entraced (deep ca
Message: foo
Backtrace:
 1. testthat::expect_is(f(), "foo")
 4. f()
 5. g()
 9. foo()

── 11. Error: errors thrown from a quasi-labelled argument are entraced (deep de
Message: foobar
Backtrace:
 1. f()
 2. g()
 6. foo()
 7. bar()

══ testthat results  ═══════════════════════════════════════════════════════════
[ OK: 5 | SKIPPED: 3 | WARNINGS: 3 | FAILED: 11 ]
1. Failure: Failure:1 (@tests.R#12) 
2. Failure: Failure:2a (@tests.R#16) 
3. Failure: Failure:2b (@tests.R#19) 
4. Failure: Failure:loop (@tests.R#24) 
5. Error: Error:1 (@tests.R#33) 
6. Error: Error:3 (@tests.R#47) 
7. Error: Recursion:1 (@tests.R#56) 
8. Error: errors thrown at block level are entraced (@tests.R#99) 
9. Error: errors thrown from a quasi-labelled argument are entraced (@tests.R#104) 
1. ...

